### DISCUSSION TITLE
[@types/node] utils.parseArgs() option with default value is possibly undefined

### DISCUSSION THREAD

**Discussion by NemoStein** [DefinitelyTyped/DefinitelyTyped#64868] [Issues with a @types package]
Consider this snippet
```js
const args = parseArgs({
  options: {
    hello: {
      type: 'string',
      default: 'world'
    }
  }
})

args.values.hello // string | undefined
```

In this case, the type of `args.values.hello` should be `string`, since it will default to `'world'` if not provided.

[Playground](https://www.typescriptlang.org/play?target=99&jsx=0&ssl=12&ssc=42&pln=12&pc=18#code/JYWwDg9gTgLgBAbzmAhlAzgUwIJQObpwC+cAZlBCHAOQB2EAJpgFwCuMwANtQLABQ-AMYRa6eGgJwAvMjRZcBABQJ+cOBDAcR6ZolVq4AC0ydOEXSr4GDMAJ5gWNMVGC081ADT7rTUilacMLrUAO7QnAy8VgZE+rF8RACU-Px2DnAAEiZmACr2mNJwaZgQpHAS6AB0AG4onKyYVcamEPwA9G1F+ZnZEHnpMs6ueHAAPnCstL6umAz8QA)

[a comment by an automated account is not shown]

**Comment by bakkot** [comment]
Fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70420.
